ARTCC/Low/Lands End transit Corridor.txt

```diff
@@ -0,0 +1,37 @@
+;Land's End Transit Corridor
+LETC         N049.49.17.790 W006.14.02.747 N049.49.07.539 W006.14.48.524
```

Comment:
Not sure there'd be any harm in prefixing with Land's End Transit Corridor? Would make it clearer what it is in Display Settings I think - what do you think?

Reply:
Can do - just thought it was a bit "wordy" for fitting in the Display window?

Reply:
Have just loaded it in, and it looks good to me!

![image](https://user-images.githubusercontent.com/12017284/79633455-98940300-815d-11ea-9df9-109989fba48e.png)

Also, I would move it to Misc/Stars_Lands End Transit Corridor.txt, just because ARTCC low is reserved for ATZs, CTRs, CTAs, etc., and things like gates/buffers/sector outlines are all in Stars.